test/run_test.py

```diff
@@ -1349,6 +1364,9 @@ def get_selected_tests(options) -> List[str]:
         # Exclude all mps tests otherwise
         options.exclude.extend(["test_mps", "test_metal"])
 
+    if options.xpu:
+        selected_tests = XPU_TEST
```

Comment:
```suggestion
        selected_tests = XPU_TEST
      else:
        # Exclude XPU tests otherwise
        options.exclude.extend(XPU_TEST)
```

Reply:
Thanks @huydhn for the review. For xpu test part, there are 2 methods to trigger the tests,
- similar with `rocm`, add `TEST_WITH_XPU`, with this env var, except `XPU_BLOCKLIST`, other default selected tests will be tested. CI workflow used this method in `.ci/pytorch/test.sh` directly.
- similar with `mps`, add `--xpu` option, with this option, it will only test `XPU_TEST` list cases. For quick test of xpu cases.

So I don't add `else` in here. How do you think?

Reply:
That's probably fine if `test_xpu.py` has already been skipped by default when the XPU device is not available.  Having an `else` part here is a bit nicer because the test will not even be scheduled, but not a major issue at all.